core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
I assumed this was some kind of golden file test, where the output is intended to match exactly this file. Is this not what the output would be with this change?